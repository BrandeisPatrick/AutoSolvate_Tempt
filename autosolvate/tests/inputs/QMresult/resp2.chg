 -0.006817  0.023890 -0.089839 -0.086695 -0.067864 -0.097861  0.036703  0.037964
 -0.099709 -0.071016 -0.101301 -0.065938  0.035198  0.124750  0.164080  0.170896
  0.147513  0.149657  0.167343  0.161415  0.128853  0.031670 -0.072326 -0.093413
 -0.063793 -0.103726  0.029571  0.033963 -0.088487 -0.071721 -0.099766 -0.085487
  0.030087  0.126165  0.167842  0.169728  0.144883  0.152373  0.167007  0.161396
  0.122921  0.039468 -0.062794 -0.090534 -0.069199 -0.106789  0.027880  0.039424
 -0.087547 -0.073689 -0.102892 -0.068034  0.039707  0.117370  0.166393  0.170036
  0.141761  0.154857  0.167580  0.160236  0.116657
