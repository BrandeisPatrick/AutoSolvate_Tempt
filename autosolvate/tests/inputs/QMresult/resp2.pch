
Resp charges for organic molecule                                               

iqopt   irstrnt  ihfree     qwt
  2      1      1        0.001000

 rel.rms   dipole mom       Qxx      Qyy      Qzz
    0.00505   0.02501  20.60223  21.37429 -41.97652

          Point charges before & after optimization
    NO   At.No.    q0           q(opt)   IVARY  d(rstr)/dq 
     1    26  -0.006817      -0.006817  -99    0.009977
     2     7   0.023890       0.023890  -99    0.009726
     3     6  -0.089839      -0.089839  -99    0.007439
     4     6  -0.086695      -0.086695  -99    0.007556
     5     6  -0.067864      -0.067864  -99    0.008274
     6     6  -0.097861      -0.097861  -99    0.007147
     7     6   0.036703       0.036703  -99    0.009388
     8     6   0.037964       0.037964  -99    0.009349
     9     6  -0.099709      -0.099709  -99    0.007081
    10     6  -0.071016      -0.071016  -99    0.008153
    11     6  -0.101301      -0.101301  -99    0.007025
    12     6  -0.065938      -0.065938  -99    0.008348
    13     7   0.035198       0.035198  -99    0.009433
    14     1   0.124750       0.124750  -99    0.000000
    15     1   0.164080       0.164080  -99    0.000000
    16     1   0.170896       0.170896  -99    0.000000
    17     1   0.147513       0.147513  -99    0.000000
    18     1   0.149657       0.149657  -99    0.000000
    19     1   0.167343       0.167343  -99    0.000000
    20     1   0.161415       0.161415  -99    0.000000
    21     1   0.128853       0.128853  -99    0.000000
    22     7   0.031670       0.031670  -99    0.009533
    23     6  -0.072326      -0.072326  -99    0.008103
    24     6  -0.093413      -0.093413  -99    0.007308
    25     6  -0.063793      -0.063793  -99    0.008431
    26     6  -0.103726      -0.103726  -99    0.006941
    27     6   0.029571       0.029571  -99    0.009590
    28     6   0.033963       0.033963  -99    0.009469
    29     6  -0.088487      -0.088487  -99    0.007489
    30     6  -0.071721      -0.071721  -99    0.008126
    31     6  -0.099766      -0.099766  -99    0.007079
    32     6  -0.085487      -0.085487  -99    0.007601
    33     7   0.030087       0.030087  -99    0.009576
    34     1   0.126165       0.126165  -99    0.000000
    35     1   0.167842       0.167842  -99    0.000000
    36     1   0.169728       0.169728  -99    0.000000
    37     1   0.144883       0.144883  -99    0.000000
    38     1   0.152373       0.152373  -99    0.000000
    39     1   0.167007       0.167007  -99    0.000000
    40     1   0.161396       0.161396  -99    0.000000
    41     1   0.122921       0.122921  -99    0.000000
    42     7   0.039468       0.039468  -99    0.009302
    43     6  -0.062794      -0.062794  -99    0.008469
    44     6  -0.090534      -0.090534  -99    0.007413
    45     6  -0.069199      -0.069199  -99    0.008223
    46     6  -0.106789      -0.106789  -99    0.006835
    47     6   0.027880       0.027880  -99    0.009633
    48     6   0.039424       0.039424  -99    0.009303
    49     6  -0.087547      -0.087547  -99    0.007524
    50     6  -0.073689      -0.073689  -99    0.008050
    51     6  -0.102892      -0.102892  -99    0.006970
    52     6  -0.068034      -0.068034  -99    0.008268
    53     7   0.039707       0.039707  -99    0.009294
    54     1   0.117370       0.117370  -99    0.000000
    55     1   0.166393       0.166393  -99    0.000000
    56     1   0.170036       0.170036  -99    0.000000
    57     1   0.141761       0.141761  -99    0.000000
    58     1   0.154857       0.154857  -99    0.000000
    59     1   0.167580       0.167580  -99    0.000000
    60     1   0.160236       0.160236  -99    0.000000
    61     1   0.116657       0.116657  -99    0.000000

        Statistics of the fitting:
  The initial sum of squares (ssvpot)                     61.256
  The residual sum of squares (chipot)                     0.002
  The std err of estimate (sqrt(chipot/N))               0.00083
  ESP relative RMS (SQRT(chipot/ssvpot))                 0.00505

 Dipole Moment (Debye)=   0.02501
