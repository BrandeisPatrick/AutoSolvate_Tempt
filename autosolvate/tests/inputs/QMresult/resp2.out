
 -----------------------------------------------
      Restrained ESP Fit 2.3  Amber 4.1
 -----------------------------------------------
 Resp charges for organic molecule                                               
 -----------------------------------------------


 inopt       =     0   ioutopt     =     1
 nmol        =     1   iqopt       =     2
 ihfree      =     1   irstrnt     =     1
 iunits      =     0   qwt         =   0.00100000
 wtmol(1) =     1.000000
 subtitle:

Resp charges for organic molecule                                               
ich =   2  iuniq =  61
    1   26  -99
    2    7  -99
    3    6  -99
    4    6  -99
    5    6  -99
    6    6  -99
    7    6  -99
    8    6  -99
    9    6  -99
   10    6  -99
   11    6  -99
   12    6  -99
   13    7  -99
   14    1  -99
   15    1  -99
   16    1  -99
   17    1  -99
   18    1  -99
   19    1  -99
   20    1  -99
   21    1  -99
   22    7  -99
   23    6  -99
   24    6  -99
   25    6  -99
   26    6  -99
   27    6  -99
   28    6  -99
   29    6  -99
   30    6  -99
   31    6  -99
   32    6  -99
   33    7  -99
   34    1  -99
   35    1  -99
   36    1  -99
   37    1  -99
   38    1  -99
   39    1  -99
   40    1  -99
   41    1  -99
   42    7  -99
   43    6  -99
   44    6  -99
   45    6  -99
   46    6  -99
   47    6  -99
   48    6  -99
   49    6  -99
   50    6  -99
   51    6  -99
   52    6  -99
   53    7  -99
   54    1  -99
   55    1  -99
   56    1  -99
   57    1  -99
   58    1  -99
   59    1  -99
   60    1  -99
   61    1  -99
 new q0 values to be read  61

  ----------------------------------------------------------------------------
     ATOM                       COORDINATES                         CHARGE
                       X             Y                Z
  ----------------------------------------------------------------------------
  ----------------------------------------------------------------------------


 Charge on the molecule(ich) =    2
 Total number of atoms (iuniq)     =   61
 Weight factor on initial charge restraints(qwt)=     0.10000D-02


  there are  1 charge constraints:


     1       1
     2       1
     3       1
     4       1
     5       1
     6       1
     7       1
     8       1
     9       1
    10       1
    11       1
    12       1
    13       1
    14       1
    15       1
    16       1
    17       1
    18       1
    19       1
    20       1
    21       1
    22       1
    23       1
    24       1
    25       1
    26       1
    27       1
    28       1
    29       1
    30       1
    31       1
    32       1
    33       1
    34       1
    35       1
    36       1
    37       1
    38       1
    39       1
    40       1
    41       1
    42       1
    43       1
    44       1
    45       1
    46       1
    47       1
    48       1
    49       1
    50       1
    51       1
    52       1
    53       1
    54       1
    55       1
    56       1
    57       1
    58       1
    59       1
    60       1
    61       1

 Reading esp"s for molecule   1
 total number of atoms      =    61
 total number of esp points =  2274


 center     X       Y       Z 
    1  -0.8339361E-02   0.5570913E-02   0.5688076E-03
    2   0.9767050E+00   0.3155058E+01   0.1836640E+01
    3   0.2242468E+01   0.3301042E+01   0.4041863E+01
    4   0.2952126E+01   0.5581310E+01   0.5143715E+01
    5   0.2338168E+01   0.7828922E+01   0.3912519E+01
    6   0.1031507E+01   0.7704648E+01   0.1631155E+01
    7   0.3686667E+00   0.5349431E+01   0.6340371E+00
    8  -0.1030517E+01   0.5003186E+01  -0.1750550E+01
    9  -0.1955001E+01   0.6991337E+01  -0.3224795E+01
   10  -0.3254832E+01   0.6463355E+01  -0.5451677E+01
   11  -0.3598498E+01   0.3945595E+01  -0.6149828E+01
   12  -0.2632220E+01   0.2058562E+01  -0.4593578E+01
   13  -0.1368360E+01   0.2544116E+01  -0.2436524E+01
   14  -0.2862880E+01   0.8446698E-01  -0.5082144E+01
   15  -0.4596929E+01   0.3436699E+01  -0.7866439E+01
   16  -0.3984187E+01   0.7988855E+01  -0.6613104E+01
   17  -0.1675010E+01   0.8934737E+01  -0.2642969E+01
   18   0.5415464E+00   0.9428664E+01   0.6405264E+00
   19   0.2864829E+01   0.9644947E+01   0.4708022E+01
   20   0.3965794E+01   0.5578247E+01   0.6925633E+01
   21   0.2689473E+01   0.1521451E+01   0.4948994E+01
   22  -0.3445218E+01  -0.2378050E+00   0.1540146E+01
   23  -0.4444781E+01   0.1327083E+01   0.3283240E+01
   24  -0.6896780E+01   0.1077446E+01   0.4201122E+01
   25  -0.8404670E+01  -0.8744897E+00   0.3273472E+01
   26  -0.7396488E+01  -0.2501637E+01   0.1466974E+01
   27  -0.4914161E+01  -0.2148006E+01   0.6330223E+00
   28  -0.3659096E+01  -0.3762602E+01  -0.1259402E+01
   29  -0.4770389E+01  -0.5895503E+01  -0.2353146E+01
   30  -0.3428863E+01  -0.7289182E+01  -0.4137890E+01
   31  -0.9975392E+00  -0.6507693E+01  -0.4787198E+01
   32  -0.1782012E-02  -0.4371594E+01  -0.3619835E+01
   33  -0.1280080E+01  -0.3006193E+01  -0.1891476E+01
   34   0.1880803E+01  -0.3713344E+01  -0.4079974E+01
   35   0.1221065E+00  -0.7526902E+01  -0.6169262E+01
   36  -0.4268460E+01  -0.8950049E+01  -0.5000909E+01
   37  -0.6660164E+01  -0.6474941E+01  -0.1817529E+01
   38  -0.8539168E+01  -0.4024975E+01   0.7137023E+00
   39  -0.1032867E+02  -0.1127609E+01   0.3938562E+01
   40  -0.7594864E+01   0.2393249E+01   0.5609635E+01
   41  -0.3224549E+01   0.2822905E+01   0.3964133E+01
   42   0.1625762E+01  -0.2347503E+01   0.2461346E+01
   43   0.5151185E+00  -0.3482508E+01   0.4452223E+01
   44   0.1772081E+01  -0.5168684E+01   0.6031479E+01
   45   0.4299805E+01  -0.5724908E+01   0.5535596E+01
   46   0.5469618E+01  -0.4567107E+01   0.3481618E+01
   47   0.4096170E+01  -0.2886916E+01   0.1974465E+01
   48   0.5151384E+01  -0.1555398E+01  -0.2340293E+00
   49   0.7669028E+01  -0.1746349E+01  -0.1013162E+01
   50   0.8476357E+01  -0.4185139E+00  -0.3137420E+01
   51   0.6735267E+01   0.1075124E+01  -0.4434246E+01
   52   0.4259754E+01   0.1185257E+01  -0.3552908E+01
   53   0.3453905E+01  -0.9287437E-01  -0.1502748E+01
   54   0.2860439E+01   0.2326100E+01  -0.4517281E+01
   55   0.7271031E+01   0.2143479E+01  -0.6099845E+01
   56   0.1042583E+02  -0.5488577E+00  -0.3762585E+01
   57   0.8994006E+01  -0.2912709E+01   0.2509556E-01
   58   0.7430747E+01  -0.4978266E+01   0.3058992E+01
   59   0.5343820E+01  -0.7033893E+01   0.6720856E+01
   60   0.7775373E+00  -0.6019065E+01   0.7609523E+01
   61  -0.1447390E+01  -0.3006651E+01   0.4785473E+01
 Initial ssvpot =    61.256


 Number of unique UNfrozen centers=    0
 ALL charges are frozen!!!

 Non-linear optimization requested.
 qchnge =    0.1413601346E-01
 qchnge =     0.000000000    
 qchnge =     0.000000000    

 Convergence in    2 iterations


Resp charges for organic molecule                                               

          Point Charges Before & After Optimization

    no.  At.no.    q(init)       q(opt)     ivary    d(rstr)/dq
    1  26      -0.006817      -0.006817    -99       0.009977
    2   7       0.023890       0.023890    -99       0.009726
    3   6      -0.089839      -0.089839    -99       0.007439
    4   6      -0.086695      -0.086695    -99       0.007556
    5   6      -0.067864      -0.067864    -99       0.008274
    6   6      -0.097861      -0.097861    -99       0.007147
    7   6       0.036703       0.036703    -99       0.009388
    8   6       0.037964       0.037964    -99       0.009349
    9   6      -0.099709      -0.099709    -99       0.007081
   10   6      -0.071016      -0.071016    -99       0.008153
   11   6      -0.101301      -0.101301    -99       0.007025
   12   6      -0.065938      -0.065938    -99       0.008348
   13   7       0.035198       0.035198    -99       0.009433
   14   1       0.124750       0.124750    -99       0.000000
   15   1       0.164080       0.164080    -99       0.000000
   16   1       0.170896       0.170896    -99       0.000000
   17   1       0.147513       0.147513    -99       0.000000
   18   1       0.149657       0.149657    -99       0.000000
   19   1       0.167343       0.167343    -99       0.000000
   20   1       0.161415       0.161415    -99       0.000000
   21   1       0.128853       0.128853    -99       0.000000
   22   7       0.031670       0.031670    -99       0.009533
   23   6      -0.072326      -0.072326    -99       0.008103
   24   6      -0.093413      -0.093413    -99       0.007308
   25   6      -0.063793      -0.063793    -99       0.008431
   26   6      -0.103726      -0.103726    -99       0.006941
   27   6       0.029571       0.029571    -99       0.009590
   28   6       0.033963       0.033963    -99       0.009469
   29   6      -0.088487      -0.088487    -99       0.007489
   30   6      -0.071721      -0.071721    -99       0.008126
   31   6      -0.099766      -0.099766    -99       0.007079
   32   6      -0.085487      -0.085487    -99       0.007601
   33   7       0.030087       0.030087    -99       0.009576
   34   1       0.126165       0.126165    -99       0.000000
   35   1       0.167842       0.167842    -99       0.000000
   36   1       0.169728       0.169728    -99       0.000000
   37   1       0.144883       0.144883    -99       0.000000
   38   1       0.152373       0.152373    -99       0.000000
   39   1       0.167007       0.167007    -99       0.000000
   40   1       0.161396       0.161396    -99       0.000000
   41   1       0.122921       0.122921    -99       0.000000
   42   7       0.039468       0.039468    -99       0.009302
   43   6      -0.062794      -0.062794    -99       0.008469
   44   6      -0.090534      -0.090534    -99       0.007413
   45   6      -0.069199      -0.069199    -99       0.008223
   46   6      -0.106789      -0.106789    -99       0.006835
   47   6       0.027880       0.027880    -99       0.009633
   48   6       0.039424       0.039424    -99       0.009303
   49   6      -0.087547      -0.087547    -99       0.007524
   50   6      -0.073689      -0.073689    -99       0.008050
   51   6      -0.102892      -0.102892    -99       0.006970
   52   6      -0.068034      -0.068034    -99       0.008268
   53   7       0.039707       0.039707    -99       0.009294
   54   1       0.117370       0.117370    -99       0.000000
   55   1       0.166393       0.166393    -99       0.000000
   56   1       0.170036       0.170036    -99       0.000000
   57   1       0.141761       0.141761    -99       0.000000
   58   1       0.154857       0.154857    -99       0.000000
   59   1       0.167580       0.167580    -99       0.000000
   60   1       0.160236       0.160236    -99       0.000000
   61   1       0.116657       0.116657    -99       0.000000

 Sum over the calculated charges:      2.000

        Statistics of the fitting:
  The initial sum of squares (ssvpot)                     61.256
  The residual sum of squares (chipot)                     0.002
  The std err of estimate (sqrt(chipot/N))               0.00083
  ESP relative RMS (SQRT(chipot/ssvpot))                 0.00505

 Center of Mass (Angst.):

 X  =   0.00000      Y  =   0.00000      Z  =   0.00000

 Dipole (Debye):

 X  =   0.01251      Y  =  -0.01035      Z  =  -0.01903

 Dipole Moment (Debye)=   0.02501

 Quadrupole (Debye*Angst.):

 Qxx =  20.60223     QYY =  21.37429     QZZ = -41.97652
 Qxy =-278.30420     QXZ =-270.98453     QYZ =-286.54793
